jupyter-notebooks/analytics/user-guide/1. Getting Started with the Planet Analytics API.ipynb

```diff
@@ -400,7 +416,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Further on in this tutorial we'll cover the specifics of this response, but for now let's focus on the format of our respones."
+    "<div class=\"callout-red top-margin\">\n",
```

Comment:
It seems a bit awkward to use html with custom styles here. Can't this just use markdown like most of the rest of the notebook(s)?

Reply:
I'll go ahead and strip out all the CSS; it *does* make things look more visually appealing and clearer, but I understand how it makes the notebooks harder to maintain.

Reply:
I've removed all the CSS and extraneous HTML elements.